src/app.ts

```diff
@@ -114,15 +114,17 @@ export async function getApp(
       : {},
   )
 
-  void app.register(fastifyGracefulShutdown, {
-    resetHandlersOnInit: true,
-    timeout: GRACEFUL_SHUTDOWN_TIMEOUT_IN_MSECS,
-  })
+  if (!isDevelopment()) {
+    await app.register(fastifyGracefulShutdown, {
```

Comment:
ts-node-dev doesn't work with fastify-graceful-shutdown, and it's not useful locally anyway